graphql/introspection/introspection.go

```diff
@@ -43,7 +43,7 @@ type InputValue struct {
 	Name         string
 	Description  string
 	Type         Type
-	DefaultValue string
+	DefaultValue *string
```

Comment:
can you reference the spec doc in the commit message?

http://facebook.github.io/graphql/October2016/#sec-The-__InputValue-Type